src/diffCheckBindings.cc

```diff
@@ -20,7 +20,7 @@ PYBIND11_MODULE(diffcheck_bindings, m) {
     submodule_test.def("test", &test, "Simple function testing a vanilla python bindings.");
```

Comment:
Simple question, why do you keep a "test" module ?

Reply:
just to have a simple test component that we can import to see if everythin is alright. It's more of an utility than anything else.